marketplace_builder/views/pages/how-to/integrations/integrating-slack-api-call-notifications.liquid

```diff
@@ -0,0 +1,101 @@
+---
+converter: markdown
+metadata:
+  title: Integrating Slack Using API Call Notifications 
+slug: how-to/integrations/integrating-slack-api-call-notifications
+---
+
+This guide will help you create a Slack integration for Platform OS using API Call Notifications. 
+
+## Requirements
+To follow this tutorial, you should be familiar with creating API call notifications. You also have to have access to a Slack workspace, have enabled webhooks, and created an incoming webhook. You will need the webhook URL for creating the API call notification.  
+
+* [Creating an API Call Notification](/get-started/notifications/creating-api-call-notification)
+* [Creating an Incoming Webhook (Slack Documentation)](https://api.slack.com/incoming-webhooks#getting-started)
+
+## Steps
+
+Integrating Slack using API call notifications is a two-step process:
+
+1. Create API call notification
+2. Add API call notification to form
+
+### Step 1: Create API call notification
+
+Create an API call notification using the webhook URL:
+
+{% raw %}
+
+<pre><code class="language-liquid">
+---
+name: { name of your file here }
+delay: '0'
+enabled: true
+format: http
+headers: '{ "Content-Type": "application/json" }'
+request_type: POST
+to: { add your webhook URL here }
+trigger_condition: t
+---
+{
+	{ add your JSON payload here }
```

Comment:
Seeing all those placeholders for data/urls i think an example would be helpful with everything filled with at least some data, even if not working, but formatted correctly.